ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -311,11 +360,17 @@ private void readWithProgress(
       int requestId,
       ResponseBody responseBody) throws IOException {
     Reader reader = responseBody.charStream();
+    ProgressResponseBody progressResponseBody = (ProgressResponseBody)responseBody;
```

Comment:
@davidaurelio I ran into this as well. Did you transpose all the mocks that I added to `NetworkingModuleTest.java` in my diff to the internal test as well?